.prod/uwsgi.ini

```diff
@@ -8,3 +8,6 @@ gid = nogroup
 master = 1
 processes = 2
 threads = 2
+; don't log readiness and healtzh endpoints
```

Comment:
THERE IS A TYPO IN THIS COMMENT!!

Reply:
I just blindly copypasted it from the example. :sob:

Reply:
Fixed. :+1: